### ISSUE TITLE
Possible rename?

### ISSUE THREAD

**Issue by davedawkins** [AngelMunoz/SutilWebComponents#1]
Hi Angel,
my only suggestion at this point is to rename this repo "sutil-shoelace" (and the nuget package). We can always create other packages for other web component libraries as they appear. This way, we only make dependencies on the target library that the user wants. What do you think?
Cheers!
Dave

**Comment by AngelMunoz** [comment]
Hey, It can be possible indeed I was thinking of following a structure like this 
https://github.com/fable-compiler/fable-browser/tree/master/src
(after creating a couple of packages) and if needed move it to a Sutil org in the case it ever gets to exist 
So the repo would be something like this (but not necessarily every package)

- packages
    - Shoelace
    - [Crayons](https://crayons.freshworks.com/)
    - [Curvenote](https://curvenote.dev/svg)
    - [Kor](https://kor-ui.com/introduction/welcome)
    - Lion
    - Material

every package would have it's own NuGet Package but is also a little bit challenging since there are packages like Lion and Material that provide each component separately, so it also can be something like this

- packages
    - Shoelace
    - [Crayons](https://crayons.freshworks.com/)
    - [Curvenote](https://curvenote.dev/svg)
    - [Kor](https://kor-ui.com/introduction/welcome)
    - [Lion.Button](https://lion-web.netlify.app/components/interaction/button/overview/#button-overview)
    - [Lion.Select](https://lion-web.netlify.app/components/inputs/select-rich/overview/#select-rich-overview)
    - [Material.Drawer](https://github.com/material-components/material-components-web-components/tree/master/packages/drawer)
    - [Material.Button](https://github.com/material-components/material-components-web-components/tree/master/packages/button)

I think it would be nice to have a "starter" and then leave it  in one place to try and make it a community effort and it should make contributions somewhat more clear You wouldn't need to provide a **full** package like shoelace, perhaps someone just needs a material component, they can contribute it and be ready to go

How does that sound?

**Comment by AngelMunoz** [comment]
It also could be a little cumbersome to create a whole package for just a simple element but Fable Browser has been working like that for a bit, perhaps @alfonsogarciacaro might have some insights on how difficult or expensive is to maintain the structure of fable-browser if he may of course 😁

**Comment by alfonsogarciacaro** [comment]
Yes, it's a tricky balance. Separating things in different repos, packages adds more complexity but gives you more flexibility. Fro Fable.Browser you can read about the motivation here: https://fable.io/blog/Announcing-2-2.html#fable-browser

Basically it was about making it easier to get maintainers. It's difficult to convince someone to maintain the whole Fable.Browser repo but if they're interested in say WebRC they can become owners of that specific package :smile: It also avoids problems with versioning.

It's true it's cumbersome to have different packages for each component, particularly for the user. But it they correspond to different npm packages maybe that's the way to go, Fulma did the same with the extra Bulma components. You can also use Femto metadata so users automatically install the corresponding npm package.

**Comment by davedawkins** [comment]
The one-to-one relationship to npm package is exactly what I was thinking of. `SutilWebComponents` has the potential to bloat, without a well-defined boundary. `sutil-shoelace` makes it clear what the scope is.

**Comment by davedawkins** [comment]
Overall I like the pla. I'm not yet sure what to suggest regarding Lion and Material. Sometimes it's easier to see what people decide they need and then go from there.

**Comment by AngelMunoz** [comment]
> The one-to-one relationship to npm package is exactly what I was thinking of. SutilWebComponents has the potential to bloat, without a well-defined boundary. sutil-shoelace makes it clear what the scope is.

I might have miss communicated there,  I want to think of `SutilWebComponents` as a central hub for web component wrapper libraries, each project/directory is a clearly defined package and it's tied to a single npm package in the case of  shoelace it's exclusive for shoelace when some one else adds another project it should be clearly defined that is a self contained to it's types and npm package

I guess once I'm further down the road I'll add the contributing to explaining those steps

but in short I don't want to publish a `SutilWebComponents` nuget package, I'd rather have them to be`Sutil.Shoelace`, `Sutil.Material.Button` and so on

> Overall I like the pla. I'm not yet sure what to suggest regarding Lion and Material. Sometimes it's easier to see what people decide they need and then go from there.

Yeah those were just some examples of what could be in the repository I'll be contributing myself one or two other libraries that I'm interested in then if anyone else wants something else we'll see what can be done in that moment
